test/common/routes_exist.go

```diff
@@ -0,0 +1,220 @@
+package common
+
+import (
+	"fmt"
+	"testing"
+
+	goctx "context"
+
+	routev1 "github.com/openshift/api/route/v1"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type expectedRoute struct {
+	Name            string
+	isTLS           bool
+	ServiceName     string
+	IsGeneratedName bool
+}
```

Comment:
Looks like people are starting to put all the types in `types.go` so just would be good for consistency :thinking: 
https://github.com/integr8ly/integreatly-operator/blob/master/test/common/types.go